include/metaverse/explorer/extensions/commands/addnode.hpp

```diff
@@ -73,7 +73,17 @@ class addnode: public command_extension
             "ACCOUNTAUTH",
             value<std::string>(&auth_.auth)->required(),
             BX_ACCOUNT_AUTH
-	    );
+	    )
+        (
+            "address,a",
+            value<std::string>(&argument_.address),
+            "The target node address[x.x.x.x:port]."
+        )
+        (
+            "command,c",
```

Comment:
addnode has function of remove is strange.

Reply:
remove the "remove node" function.